authui/src/authflowv2/components/phone-input.css

```diff
@@ -103,6 +103,42 @@
     box-shadow: var(--phone-input__dropdown__shadow);
   }
 
+  .phone-input__search {
+    @apply relative;
+    @apply flex;
+    @apply flex-row;
+    @apply items-center;
+
+    border-bottom: 1px solid var(--color-grey-100);
+  }
+
+  .phone-input__search-input {
```

Comment:
Can we reuse the input field component here?

Reply:
The search input's style is pretty different from the input component, or you meant adding a variant to input?